docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -7,38 +7,40 @@
 PWM and DSHOT Control with a Flight Controller
 ***********************************************
 
-This tutorial is meant to walk you through the process of doing basic setup and testing for controlling a Vertiq motor
-with a flight controller. This covers only the basics of testing that the flight controller can make the motor spin
-with the appropriate protocol, not the setup of any additional flight controller peripherals like an RC controller
-or a GPS. This tutorial should take you from a totally fresh motor and flight controller, to a motor that can be controlled
-in Voltage mode with the PWM protocol or the DSHOT protocols from the flight controller.
+This tutorial is meant to walk you through the process of performing basic setup and testing for controlling a Vertiq module
+with a flight controller using :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`DSHOT <hobby_dshot>`. This tutorial covers only the basics of testing 
+that the flight controller can make the module spin with the appropriate protocol, not the setup of any additional flight controller peripherals like an RC controller
+or a GPS. This tutorial should take you from a totally fresh module and flight controller, to a module that can be controlled
+with the :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`DSHOT <hobby_dshot>` protocols from the flight controller.
 
-This tutorial covers setup and testing on **Ardupilot using Mission Planner** and **PX4 using QGroundControl**. It covers using
-the **PWM** or **DSHOT** protocol, since the setup process is largely similar for both protocols. In the places where there are differences,
-the proper steps for both types of protocol have been recorded.
+This tutorial covers setup and testing on `ArduCopter <https://ardupilot.org/copter/>`_ using `Mission Planner <https://ardupilot.org/planner/>`_ and 
+`PX4 <https://px4.io/>`_ using `QGroundControl <http://qgroundcontrol.com/>`_. It covers using both the :ref:`Standard PWM <hobby_standard_pwm>` additonal :ref:`DSHOT <hobby_dshot>` 
+protocols, since the setup process is largely similar for both protocols. In the places where there are differences, the proper steps for both types of protocol are explained.
 
-These instructions are applicable to both the **Vertiq 8108** and the **Vertiq 2306** with speed firmware. Though they have
```

Comment:
Before we only had the two types of modules, so I explicitly talked about both. Now with ever increasing numbers of modules, I am just focusing on the 8108 and pointing out that these instructions should work for any module. Removed 2306 references.